index.js

```diff
@@ -11,241 +11,352 @@ inputarea.sortable({
+          formatsCommand.push('106')
+          formatsExample.push('color-bg-light-cyan')
           break
         case 'White':
-          formats.push('107')
+          formatsCommand.push('107')
+          formatsExample.push('color-bg-white')
           break
         default:
           command += '\\[\\e[48;5;' + $(this).attr('data-bg-color') + 'm\\]'
+          // Use given color
+          colorFG = $(this).attr('data-bg-color')
       }
+    } else {
+      formatsExample.push('color-bg-black')
     }
 
-    if (formats.length > 0) {
-      command += '\\[\\e[' + formats.join(';') + 'm\\]'
-
-      if (formats.length === 1 && formats[0] === '0') {
+    if (formatsCommand.length > 0) {
+      command += '\\[\\e[' + formatsCommand.join(';') + 'm\\]'
+      if (formatsExample.indexOf('color-'))
+      colorFG = (colorFG === null ? '' : `style="color:${parseColor(colorFG)};"`)
+      colorBG = (colorBG === null ? '' : `style="color:${parseColor(colorBG)};"`)
+      example += `<span class="${formatsExample.join(' ')}"${colorFG}${colorBG}>`
+      if (formatsCommand.length === 1 && formatsCommand[0] === '0') {
         requireReset = false
       } else {
         requireReset = true
       }
     } else {
       requireReset = false
+      example += '<span>'
     }
 
     switch ($(this).html().replace(/\s*<i.*<\/i>/gi, '').trim()) {
       case 'Username':
         command += '\\u'
+        example += 'Scriptim'
         break
       case 'Hostname (short)':
         command += '\\h'
+        example += 'Scriptim-PC'
         break
       case 'Hostname':
         command += '\\H'
+        example += 'Scriptim-PC'
         break
       case 'Shell Name':
         command += '\\s'
+        example += 'bash'
         break
       case 'Bash Version':
         command += '\\v'
+        example += '2.0'
         break
       case 'Bash Release':
         command += '\\V'
+        example += '2.00.0'
         break
       case 'Terminal':
         command += '\\l'
+        example += '2'
         break
       case 'Working Directory':
         command += '\\w'
+        example += '~/Desktop'
         break
       case 'Working Directory (basename)':
         command += '\\W'
+        example += 'Desktop'
         break
       case 'Date':
         command += '\\d'
+        example += 'Tue May 26'
         break
       case 'Date (formatted)':
         command += '\\D{' + ($(this).attr('data-dateformat') ? $(this).attr('data-dateformat') : '') + '}'
+        example += '16:22:18'
         break
       case 'Time (24h)':
         command += '\\t'
+        example += '16:22:18'
         break
       case 'Time (12h)':
         command += '\\T'
+        example += '04:22:18'
         break
       case 'Time (am/pm)':
         command += '\\@'
+        // No idea where the space comes from, in my bash it is there...
+        example += '04:22 '
         break
       case 'Time (w/o seconds)':
         command += '\\A'
+        example += '16:22'
```

Comment:
It would be nice if the actual time would be inserted here.

In particular regarding *Date (formatted)*, the entered date format should be respected.
The example for *Time (am/pm)* should actually be either `04:22 AM` or `04:22 PM`.